js/app.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const oReqVader = new XMLHttpRequest();
+oReqVader.addEventListener('load', reqListenerVaderName);
+oReqVader.open('GET', 'https://swapi.co/api/people/4/');
+oReqVader.send();
+
+function reqListenerVaderName() {
+  let nameOf4 = document.getElementById('person4Name');
+  nameOf4.innerHTML = JSON.parse(this.responseText).name;
+}
+
+const oReqVaderHomeWorld = new XMLHttpRequest();
+oReqVaderHomeWorld.addEventListener('load', reqListenerVaderWorld);
+oReqVaderHomeWorld.open('GET', 'https://swapi.co/api/planets/1/');
+oReqVaderHomeWorld.send();
+
+function reqListenerVaderWorld() {
+  let worldOf4 = document.getElementById('person4HomeWorld');
+  worldOf4.innerHTML = JSON.parse(this.responseText).name;
+}
+
+const oReqSolo = new XMLHttpRequest();
+oReqSolo.addEventListener('load', reqListenerSoloName);
+oReqSolo.open('GET', 'https://swapi.co/api/people/14/');
+oReqSolo.send();
+
+function reqListenerSoloName() {
+  let nameOf14 = document.getElementById('person14Name');
+  nameOf14.innerHTML = JSON.parse(this.responseText).name;
+}
+
+const oReqSoloSpecies = new XMLHttpRequest();
+oReqSoloSpecies.addEventListener('load', reqListenerSoloSpecies);
+oReqSoloSpecies.open('GET', 'https://swapi.co/api/species/1/');
+oReqSoloSpecies.send();
```

Comment:
Getting the person's species should be done from within the `reqListenerSoloName`